common/templates/mathjax_include.html

```diff
@@ -1,10 +1,22 @@
 ## mako
 ## File:   templates/mathjax_include.html
-##
-## Advanced mathjax using 2.0-latest CDN for Dynamic Math
-##
-## This enables ASCIIMathJAX, and is used by js_textbox
 
+<%page args="disable_fast_preview=True"/>
```

Comment:
Nit: it is confusing that this is in common, but it is only used by LMS. It may be worth a comment with a pointer to the same setting in the CMS require-config.js file. I suppose you could also move this into LMS so it is clearer that it is not truly a common template.